diff --git a/node_modules/next-prisma-plugin/dist/index.js b/node_modules/next-prisma-plugin/dist/index.js
index 830e199..d4df8ec 100644
--- a/node_modules/next-prisma-plugin/dist/index.js
+++ b/node_modules/next-prisma-plugin/dist/index.js
@@ -40,7 +40,7 @@ var withPrismaPlugin = function (nextConfig) {
                 webpack: function (config, options) {
                     var ignore = ['.prisma/client', '@prisma/client'];
                     // const includes = ignore.map(module => (new RegExp(`${module}(?!.*node_modules)`)));
-                    var excludes = [new RegExp("node_modules(?!/(" + ignore.join('|') + ")(?!.*node_modules))")];
+                    var excludes = [`${new RegExp("node_modules(?!/(" + ignore.join('|') + ")(?!.*node_modules))")}`];
                     var ignored = config.watchOptions.ignored.filter(function (ignored) { return ignored !== '**/node_modules/**'; }).concat(excludes);
                     return Object.assign(config, {
                         plugins: __spreadArrays(config.plugins, [new webpack_plugin_1.default()]),
